<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTMarkerListWidget</class>
 <widget class="QWidget" name="TTMarkerListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>100</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>620</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>12</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QTreeWidget" name="markerListView">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Frame</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UUID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbEntryUp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="cutlistwidget.qrc">
       <normaloff>:/pixmaps/pixmaps/uparrow_18.xpm</normaloff>:/pixmaps/pixmaps/uparrow_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbEntryDown">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="cutlistwidget.qrc">
       <normaloff>:/pixmaps/pixmaps/downarrow_18.xpm</normaloff>:/pixmaps/pixmaps/downarrow_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbEntryDelete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>24</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>60</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="cutlistwidget.qrc">
       <normaloff>:/pixmaps/pixmaps/cancel_18.xpm</normaloff>:/pixmaps/pixmaps/cancel_18.xpm</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>676</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionItemUp">
   <property name="icon">
    <iconset resource="cutlistwidget.qrc">
     <normaloff>:/pixmaps/pixmaps/uparrow_18.xpm</normaloff>:/pixmaps/pixmaps/uparrow_18.xpm</iconset>
   </property>
   <property name="text">
    <string>Move &amp;Up</string>
   </property>
   <property name="toolTip">
    <string>Move selected marker one position upward</string>
   </property>
  </action>
  <action name="actionItemDelete">
   <property name="icon">
    <iconset resource="cutlistwidget.qrc">
     <normaloff>:/pixmaps/pixmaps/cancel_18.xpm</normaloff>:/pixmaps/pixmaps/cancel_18.xpm</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="toolTip">
    <string>Remove selected marker from list</string>
   </property>
  </action>
  <action name="actionItemDown">
   <property name="icon">
    <iconset resource="cutlistwidget.qrc">
     <normaloff>:/pixmaps/pixmaps/downarrow_18.xpm</normaloff>:/pixmaps/pixmaps/downarrow_18.xpm</iconset>
   </property>
   <property name="text">
    <string>Move d&amp;own</string>
   </property>
   <property name="toolTip">
    <string>Move selected marker one position down</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="cutlistwidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionItemDown</sender>
   <signal>triggered()</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onItemDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionItemUp</sender>
   <signal>triggered()</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onItemUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionItemDelete</sender>
   <signal>triggered()</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onRemoveItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEntryUp</sender>
   <signal>clicked()</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onItemUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEntryDown</sender>
   <signal>clicked()</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onItemDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbEntryDelete</sender>
   <signal>clicked()</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onRemoveItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>markerListView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onActivateMarker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>markerListView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>TTMarkerListWidget</receiver>
   <slot>onContextMenuRequest(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onItemUp()</slot>
  <slot>onItemDown()</slot>
  <slot>onRemoveItem()</slot>
  <slot>onActivateMarker()</slot>
  <slot>onContextMenuRequest(QPoint)</slot>
 </slots>
</ui>
